src/drivers/driver_zephyr.c

```diff
@@ -905,6 +905,16 @@ static void wpa_drv_zep_event_mgmt_rx(struct zep_drv_if_ctx *if_ctx,
 	wpa_supplicant_event_wrapper(if_ctx->supp_if_ctx, EVENT_RX_MGMT, &event);
```

Comment:
Please do not leave empty commit messages but write a proper text there, it is not enough to just have commit subject.

Reply:
Added commit message for each commit. Thanks!